packages/arb-token-bridge-ui/src/util/RetryableUtils.ts

```diff
@@ -33,3 +47,37 @@ export async function getRetryableTicket({
 
   return retryableTicket
 }
+
+export const getRetryableTicketExpiration = async ({
+  l1TxHash,
+  l1Provider,
+  l2Provider
+}: GetRetryableTicketExpirationParams): Promise<RetryableTicketExpirationResponse> => {
+  let daysTillExpiry: number = 0
+  let isValid = false // daysTillExpiry still loading...
+  let expiryTimestamp = 0
+
+  try {
+    const depositTxReceipt = await l1Provider.getTransactionReceipt(l1TxHash)
+    const l1TxReceipt = new L1TransactionReceipt(depositTxReceipt)
+    const l1ToL2Msg = await l1TxReceipt.getL1ToL2Message(l2Provider)
+
+    const now = dayjs()
+    const expiryDate = await l1ToL2Msg.getTimeout()
+
+    const expiryTimestamp = +expiryDate.toString() * 1000
```

Comment:
Can we do this instead of using the `+` operator:

```suggestion
const expiryTimestamp = Number(expiryDate) * 1000
```

Reply:
Done